webapp/src/main/webapp/config/listViewConfig-authorInAuthorship.xml

```diff
@@ -10,75 +10,101 @@
         PREFIX bibo: &lt;http://purl.org/ontology/bibo/&gt;
         PREFIX vitro: &lt;http://vitro.mannlib.cornell.edu/ns/vitro/0.7#&gt;
         PREFIX foaf: &lt;http://xmlns.com/foaf/0.1/&gt;
+        PREFIX vcard:  &lt;http://www.w3.org/2006/vcard/ns#&gt;
+        PREFIX obo: &lt;http://purl.obolibrary.org/obo/&gt;
 
-        SELECT DISTINCT ?subclass
+        SELECT DISTINCT
             ?authorship
             ?infoResource ?infoResourceName
-            ?dateTime
-            ?journal
-            ?volume
-            ?startPage
-            ?endPage
-            ?doi
-            ?pmid
-            ?isbn10
-            ?isbn13
-            ?publisher
-            ?locale
-            ?appearsIn
-            ?partOf
-            ?editor
-            ?hideThis
+            (sample(?irsubclass) as ?subclass)
```

Comment:
Yes, but VIVO will only use the first result anyway without tricks like the group_concat I'm using here. In order to use group_concat the other variables need to be included in the GROUP BY statement or limited to one with SAMPLE. Otherwise a " Non-group key variable in SELECT" error is thrown.